roles/nfs_file_share/tasks/nfs_file_share.yml

```diff
@@ -19,7 +19,9 @@
   ignore_errors: yes
 
 - name: Create NFS export
-  shell: '/usr/lpp/mmfs/bin/mmnfs export add {{ scale_protocols.mountpoint }}/{{ item.item }} --client "{{ compute_subnet_cidr }}(Access_Type=RW,SQUASH=no_root_squash)"'
+  shell: |
+    /usr/lpp/mmfs/bin/mmnfs export add {{ scale_protocols.mountpoint }}/{{ item.item }} --client "{{ compute_subnet_cidr }}(Access_Type=RW,SQUASH=no_root_squash)"
+    sleep 10              #This is temporary fix for Dbus error.
```

Comment:
I can suggest to add new task for sleep wait time just after this shell execution.
```
- name: Sleep for 10 seconds and continue with play
  wait_for:
    timeout: 10
```

Reply:
@jayeshh123 Thanks. we also have option to keep complete block in the wait_for but now i am considering your change.